### DISCUSSION TITLE
[Request] Change round argument in Ergast API to event_number

### DISCUSSION THREAD

**Discussion by harish5p** [theOehrly/Fast-F1#366] [General]
Also rename round in the dataframe columns to event_number. Python is recognizing round() function when using it to get a cell value

eg. `race_names_df[race_names_df['raceName'] == event]['round'].values[0]` works but `race_names_df[race_names_df['raceName'] == event].round.values[0]` doesn't work

**Comment by theOehrly** [comment]
I'm aware of that problem. The issue is that this would make it the only argument and returned value in the Ergast API wrapper that is named differently than it is in the original Ergast API. This will lead to confusion again. So the question is, where do you want to create the confusion.

It is unlucky that round is an internal name in Python. I don't really want to change this. But I could maybe add a warning to the ``.round`` function of the DataFrame, telling you that you are in fact calling the function instead of accessing the column. Also, I'm not very much a fan of using the ``.columnName`` accessor over the ``['columnName']`` variant, exactly because it can collide with existing function names like in this case.
